bin/preview-deploy/aws.user-data.sh

```diff
@@ -100,16 +100,24 @@ npm i -g yarn@1.22.18
 # Clone from Github
 git clone --single-branch -b __GIT_BRANCH__ https://github.com/CMSgov/eAPD.git
 # Build the web app and move it into place
-cd eAPD/web
-yarn add webpack@5.70.0 webpack-cli@4.9.2
-yarn install --frozen-lockfile
-API_URL=/api TEALIUM_TAG="__TEALIUM_TAG__" OKTA_DOMAIN="__OKTA_DOMAIN__" OKTA_SERVER_ID="__OKTA_SERVER_ID__" OKTA_CLIENT_ID="__OKTA_CLIENT_ID__" yarn build
+cd eAPD
+yarn install --frozen-lockfile --production=true
+
+cd web
+# yarn add webpack@5.70.0 webpack-cli@4.9.2
+WEB_ENV="dev" API_URL=/api TEALIUM_TAG="__TEALIUM_TAG__" OKTA_DOMAIN="__OKTA_DOMAIN__" OKTA_SERVER_ID="__OKTA_SERVER_ID__" OKTA_CLIENT_ID="__OKTA_CLIENT_ID__" yarn build
+
 mv dist/* /app/web
-cd ~
+
+# move over node modules
+cd ~/eAPD
+mkdir -p /app/node_modules
+mv ~/eAPD/node_modules/* /app/node_modules
+
 # Move the API code into place, then go set it up
-mv eAPD/api/* /app/api
+mv ~/eAPD/api/* /app/api
 cd /app/api
-yarn install --frozen-lockfile --production=true
+
```

Comment:
Is there a yarn install replacing this one or is it not required?

Reply:
we only need to do yarn install in one place now and not in each directory